--- a/acinclude.m4	2024-10-24 18:25:13.859572010 +0000
+++ b/acinclude.m4	2024-10-24 18:25:47.176976012 +0000
@@ -1155,7 +1155,7 @@
 AC_RUN_IFELSE([AC_LANG_SOURCE([[
 #include <time.h>

-main() {
+int main() {
 char buf[27];
 struct tm t;
 time_t old = 0;
@@ -1171,7 +1171,7 @@
 ],[
   AC_RUN_IFELSE([AC_LANG_SOURCE([[
 #include <time.h>
-main() {
+int main() {
   struct tm t, *s;
   time_t old = 0;
   char buf[27], *p;
@@ -1471,7 +1471,7 @@
 dnl
 AC_DEFUN([PHP_AC_BROKEN_SPRINTF],[
   AC_CACHE_CHECK(whether sprintf is broken, ac_cv_broken_sprintf,[
-    AC_RUN_IFELSE([AC_LANG_SOURCE([[main() {char buf[20];exit(sprintf(buf,"testing 123")!=11); }]])],[
+    AC_RUN_IFELSE([AC_LANG_SOURCE([[int main() {char buf[20];exit(sprintf(buf,"testing 123")!=11); }]])],[
       ac_cv_broken_sprintf=no
     ],[
       ac_cv_broken_sprintf=yes
@@ -1495,7 +1495,7 @@
   AC_CACHE_CHECK(whether snprintf is broken, ac_cv_broken_snprintf,[
     AC_RUN_IFELSE([AC_LANG_SOURCE([[
 #define NULL (0L)
-main() {
+int main() {
   char buf[20];
   int res = 0;
   res = res || (snprintf(buf, 2, "marcus") != 6);
@@ -1931,7 +1931,7 @@
   if test "$found" = "yes"; then
     ac_libs=$LIBS
     LIBS="$LIBS -l$2"
-    AC_RUN_IFELSE([AC_LANG_SOURCE([[main() { return (0); }]])],[found=yes],[found=no],[found=no])
+    AC_RUN_IFELSE([AC_LANG_SOURCE([[int main() { return (0); }]])],[found=yes],[found=no],[found=no])
     LIBS=$ac_libs
   fi

@@ -2679,7 +2679,7 @@

 X
     cat >conftest.$ac_ext <<X
-main()
+int main()
 {
   exit(0);
 }
@@ -2921,7 +2921,7 @@

 #define TEXT "This is the test message -- "

-main()
+int main()
 {
   int n;
