From 45b4368d5c2354781a7b2f1cad86402658868e1d Mon Sep 17 00:00:00 2001
From: Dmitry Stogov <dmitry@zend.com>
Date: Thu, 27 Feb 2020 12:27:22 +0300
Subject: [PATCH] Fixed incorrect behavior of internal memory debugger

---
 Zend/zend_alloc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Zend/zend_alloc.c b/Zend/zend_alloc.c
index 0ccc004e14..25216898ed 100644
--- a/Zend/zend_alloc.c
+++ b/Zend/zend_alloc.c
@@ -2103,6 +2103,7 @@ static zend_long zend_mm_find_leaks(zend_mm_heap *heap, zend_mm_chunk *p, uint32
 			}
 		}
 		p = p->next;
+		i = ZEND_MM_FIRST_PAGE;
 	} while (p != heap->main_chunk);
 	return count;
 }
-- 
2.32.0 (Apple Git-132)

