diff --git a/Cargo.toml b/Cargo.toml
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,11 +1,11 @@
 [package]
 name = "libddwaf-sys"
 description = "Low-level bindings to DataDog/libddwaf"
-authors.workspace = true
-version.workspace = true
-edition.workspace = true
-rust-version.workspace = true
-license.workspace = true
-repository.workspace = true
-categories.workspace = true
-keywords.workspace = true
+authors = ["DataDog, Inc. <support@datadoghq.com>"]
+version = "2.0.1"
+edition = "2021"
+rust-version = "1.85"
+license = "Apache-2.0"
+repository = "https://github.com/DataDog/libddwaf-rust"
+categories = ["api-bindings", "security"]
+keywords = ["datadog", "security", "waf", "rasp"]
